packages/components/radio/src/radio-group.vue

```diff
@@ -7,7 +7,20 @@
     :aria-label="!isLabeledByFormItem ? ariaLabel || 'radio-group' : undefined"
     :aria-labelledby="isLabeledByFormItem ? formItem!.labelId : undefined"
   >
-    <slot />
+    <slot>
+      <el-radio
+        v-for="(item, index) in props.options"
+        :key="item.key ?? index"
+        v-bind="{
+          ...item,
+          ...getOptionProps(item),
+        }"
+      >
```

Comment:
```suggestion
        v-for="(item, index) in props.options"
        :key="index"
        v-bind="getOptionProps(item)"
      >
```
Please consistent with #21545